pdf-structure.ts

```diff
@@ -359,7 +363,7 @@ module PDFStructure {
       return null
     }
     var sectionNumber = parseInt(ms[1])
-    var subsectionNumber = ms[2] != null ? parseInt(ms[2]) : null
+    var subsectionNumber = ms[2] != null && ms[2].length > 0 ? parseInt(ms[2]) : null
```

Comment:
Not a blocker, we we should specify the `radix` when calling `parseInt(x, 10)` as to avoid any ambiguity:

> If radix is undefined or 0 (or absent), JavaScript assumes the following:
If the input string begins with "0x" or "0X", radix is 16 (hexadecimal) and the remainder of the string is parsed.
If the input string begins with "0", radix is eight (octal) or 10 (decimal).  Exactly which radix is chosen is implementation-dependent.  ECMAScript 5 specifies that 10 (decimal) is used, but not all browsers support this yet.  For this reason always specify a radix when using parseInt.
If the input string begins with any other value, the radix is 10 (decimal).